src/zeit/push/interfaces.py

```diff
@@ -244,6 +247,34 @@ class MobileButtonsSource(zeit.cms.content.sources.XMLSource):
 MOBILE_BUTTONS_SOURCE = MobileButtonsSource()
 
 
+class PayloadTemplateSource(zc.sourcefactory.basic.BasicSourceFactory):
+
+    # For testing purposes, this should not be cached at all
+    # @CONFIG_CACHE.cache_on_arguments()
+    def getValues(self):
+        payload_path = zope.app.appsetup.product.getProductConfiguration(
+            'zeit.push').get('push-payload-templates')
+        template_folder = zeit.cms.interfaces.ICMSContent(
+            "http://xml.zeit.de/{}".format(payload_path))
```

Comment:
Wir könnten auch die ganze uniqueId des Ordners in die Konfig tun, nicht nur den Pfad, dann muss man hier nicht rumbasteln.